server/controllers/comments/addComments.js

```diff
@@ -0,0 +1,50 @@
+const {
+  addCommentsQuery,
+  getKindergartenId,
+} = require('../../database/queries');
+const { boomify } = require('../../utils');
+const { addCommentSchema } = require('../../utils/validation');
+const { kindergartenIdSchema } = require('../../utils/validation');
+
+const addComments = async (req, res, next) => {
+  try {
+    const { kindergartenId } = await kindergartenIdSchema.validate(req.params, {
+      abortEarly: false,
+    });
+    const { userName, comment, rating } = await addCommentSchema.validate(
+      req.body,
+      {
+        abortEarly: false,
+      }
+    );
```

Comment:
Instead of using multiple validation schema, you can use one schema to validate multiple values

```suggestion
    const { kindergartenId } = req.params;
    const { userName, comment, rating } = req.body;
    await addCommentSchema.validate(
      { userName, comment, rating, kindergartenId },
      req.params,
      {
        abortEarly: false,
      }
    );
```